From 3562cb865b13d6f59e5c3b472d073ad6bbc06ceb Mon Sep 17 00:00:00 2001
From: Julien Cristau <jcristau@debian.org>
Date: Fri, 2 Aug 2013 23:40:55 +0200
Subject: [PATCH] test: include dix-config.h in hashtabletest.c

Missing _XSERVER64 define caused inconsistent sizeof(XID) between the
test and hashtable code, leading to test failures on 64bit big endian
archs like s390x or ppc64.
---
 test/hashtabletest.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/test/hashtabletest.c b/test/hashtabletest.c
index 64c7091..6af14a8 100644
--- a/test/hashtabletest.c
+++ b/test/hashtabletest.c
@@ -1,3 +1,7 @@
+#ifdef HAVE_DIX_CONFIG_H
+#include <dix-config.h>
+#endif
+
 #include <misc.h>
 #include <stdlib.h>
 #include <stdio.h>
-- 
1.7.10.4

